### ISSUE TITLE
Tray Icon in i3wm

### ISSUE THREAD

**Issue by taigrr** [keybase/client#11995]
Keybase GUI Version: 1.0.49-20180518165324+876298b69d

The tray icon auto-closes when hovering over another window (instead of when clicking off or something) but in i3 the tray opens in the middle of the page instead of just above the tray icon, so you can't move your mouse to the tray without it closing first, unless no windows on the desktop are open.

**Comment by taigrr** [comment]
Workaround: click on the tray icon and hold until your mouse is over the popup. Not the most intuitive, but then again I am using i3 :stuck_out_tongue: 
I'll leave this open anyway and let the dev team decide whether or not to resolve it.

**Comment by Mikaela** [comment]
I think this may be a duplicate of https://github.com/keybase/client/issues/8232 which offers workaround

> (On the user's end, if they are using i3, they can fix it by adding the line `for_window [title="tracker - .*"] floating enable`.)

**Comment by kyrias** [comment]
No, #8232 just causes the windows to render weirdly because they're not floated, whereas the problem here is that the pop-up window that is floated is automatically closed when hovering over any other window.
